MdePkg/Include/IndustryStandard/PcieDoeCapbility.h

```diff
@@ -0,0 +1,82 @@
+/** @file
+  PCIe DOE Capability structure definitions
+
+  Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef PCIE_DOE_CAPBILITY_H_
+#define PCIE_DOE_CAPBILITY_H_
+
+#include <IndustryStandard/PciExpress21.h>
+
+//
+// The Data Object Exchange PCI Express Extended Capability definitions.
+// Based on section x.x.x of PCI Express Base Specification x.x.

```

Comment:
Which spec version is this based on?  Why is this in an independent file if it is part of PCI Express Specification content?

Reply:
DOE is introduced in 6.0 and this is based on the PCIe 6.0.1 spec

Reply:
It's a separate file as that keeps it self contained and easy to read

Reply:
If you want I can squash it into the `MdePkg/Include/IndustryStandard/PciExpress60.h` file

Reply:
Please squash into PciExpress60.h.

And please fill in the spec reference information